### PR TITLE
Add support for python2-pip on redhat 7

### PR DESCRIPTION
**Pull request by covataamos**

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -74,12 +74,23 @@
...
+        '6': {
+          $version       = '2.6'
+          $pip           = 'python-pip'
+        }
+        '7': {
+          $version       = '2.7'
+          $pip           = 'python2-pip'
+        }
+        default: {
+          fail("redhat version \"${::operatingsystemmajrelease}\" not supported")
```
Comment (jdavisp3) [inline manifests/params.pp]: I don't know RedHat too well -- is it safe to assume versions less than 6 won't support `pip`?

**Comment by jdavisp3** [comment]
Thanks for your contribution!

File: manifests/params.pp (hunk 1 above)
Comment (covataamos) [inline manifests/params.pp]: I'm not quiet familiar with RedHat either. I think you are right and we can expect older versions to still support `pip`, but I don't know for sure. Let me try to fire up a CentOS 5 instance on EC2 and get back to you.

Comment (covataamos) [inline manifests/params.pp]: Update: looking at the package index at https://dl.fedoraproject.org/pub/epel/5/x86_64/ I don't see python-pip there. On later releases this is where the package is fetched. So I'd say that the code is actually correct in not supporting releases lower than 6.

Comment (jdavisp3) [inline manifests/params.pp]: Makes sense!

**Comment by jdavisp3** [comment]
Thanks @covataamos, I'll make a new release later this week.

**Pull request merged**